src/components/Inputs/TextInput/TextInput.stories.tsx

```diff
@@ -164,6 +167,12 @@ Text_Input.args = {
   name: 'myTextInput',
   numbersOnly: false,
   placeholder: 'Placeholder text',
+  readonly: {
+    clearable: false,
+    enabled: false,
```

Comment:
Is it worthwhile to separate the boolean from the additional props to avoid the type checking?